From 253f076438ce72ede486a1bda55d2e4661337b70 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Mon, 17 Oct 2016 15:17:43 +0800
Subject: [PATCH 1238/5242] MLK-13387-3 regulator: consumer: add new event
 macro

commit  7379040acc503f4e358cce32ed853de8af1dbecc from
https://source.codeaurora.org/external/imx/linux-imx.git

Add one new regulator events macro 'REGULATOR_EVENT_AFT_DO_ENABLE'.
1.8v of imx7d pcie phy, should be turned on after
the 1p0d(1.0v) of pcie phy is turned on.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/regulator/core.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/regulator/core.c b/drivers/regulator/core.c
index 83a8c9b..8d95791 100644
--- a/drivers/regulator/core.c
+++ b/drivers/regulator/core.c
@@ -2203,6 +2203,7 @@ static int _regulator_do_enable(struct regulator_dev *rdev)
 	_regulator_enable_delay(delay);
 
 	trace_regulator_enable_complete(rdev_get_name(rdev));
+	_notifier_call_chain(rdev, REGULATOR_EVENT_AFT_DO_ENABLE, NULL);
 
 	return 0;
 }
-- 
1.7.9.5

